x-pack/test/functional/services/uptime/alerts.ts

```diff
@@ -14,34 +14,36 @@ export function UptimeAlertsProvider({ getService }: FtrProviderContext) {
 
   return {
     async openFlyout(alertType: 'monitorStatus' | 'tls') {
-      await testSubjects.click('xpack.uptime.alertsPopover.toggleButton', 5000);
-      await testSubjects.click('xpack.uptime.openAlertContextPanel', 5000);
+      await testSubjects.click('xpack.uptime.alertsPopover.toggleButton');
+      await testSubjects.click('xpack.uptime.openAlertContextPanel');
       if (alertType === 'monitorStatus') {
-        await testSubjects.click('xpack.uptime.toggleAlertFlyout', 5000);
+        await testSubjects.click('xpack.uptime.toggleAlertFlyout');
       } else if (alertType === 'tls') {
         await testSubjects.click('xpack.uptime.toggleTlsAlertFlyout');
       }
+      // ensure the flyout has opened
+      await testSubjects.exists('alertNameInput');
```

Comment:
This is the main fix. Everything else is mainly removing timeouts that were less than the default, or adjust the async functions to use await instead of returning since nothing needs to be returned to the caller.